modules/seo-tools/jetpack-seo-posts.php

```diff
@@ -60,4 +60,36 @@ public static function get_post_custom_description( $post ) {
 
 		return $custom_description;
 	}
-}
+
+	/**
+	 * Registers the self::DESCRIPTION_META_KEY post_meta for use in the REST API.
+	 */
+	public static function register_post_meta() {
+		if ( ! Jetpack_SEO_Utils::is_enabled_jetpack_seo() ) {
+			return;
+		}
+
+		$args = array(
+			'type' => 'string',
+			'description' => __( 'Custom post description to be used in HTML <meta /> tag.', 'jetpack' ),
+			'single' => true,
+			'default' => '',
+			'show_in_rest' => array(
+				'name' => self::DESCRIPTION_META_KEY
+			),
+		);
+
+		register_meta( 'post', self::DESCRIPTION_META_KEY, $args );
+	}
+
+	/**
+	 * Register the Advanced SEO Gutenberg extension
+	 */
+	public static function register_gutenberg_extension() {
+		if ( Jetpack_SEO_Utils::is_enabled_jetpack_seo() ) {
+			Jetpack_Gutenberg::set_extension_available( 'jetpack-seo' );
```

Comment:
Shouldn't it be named `jetpack/seo`, following how other blocks and extensions are named?

Reply:
I think the registration function in jetpack knows how to handle this and it's in line how Gutenberg's register plugin -function looks like?

Reply:
We seem to have adopted a convention to prefix our Gutenberg _blocks_ with `jetpack/`, but our Gutenberg _plugins_ with `jetpack-`. (Not that we've been super consistent about it :sweat_smile: )

Reply:
Which plugins use `jetpack-`? 

- Publicize seems to be `jetpack/publicize`
- Shortlinks is `jetpack/shortlinks`

Maybe we should stick to using `/` to avoid any confusion in the future?

Reply:
I'm indifferent how do we do now for this PR but seems like we have some janitorial work to do with plugins regardless, so I'd be okay leaving it for follow up PR:

- Move non-block integrations from `extensions/blocks` to `extensions/plugins`
- ensure all slugs are named consistently
- maaaybe refactor https://github.com/Automattic/jetpack/blob/b8717477aecd787bf7078e73a69cf1f49ada56dc/class.jetpack-gutenberg.php#L100-L112 — unless it's important for wpcom?

Reply:
and just in case it wasn't clear, I'm also up for consistency and enforcing any pattern we decide on :-)